load/load.go

```diff
@@ -71,24 +71,16 @@ func loadWithPatch(msg proto.Message, path string, fmt format.Format, patch tabl
 		// If patch file not exists, then just load from the "main" file.
 		return load(msg, path, fmt, opts)
 	}
-	var patcherr error
-	switch patch {
-	case tableaupb.Patch_PATCH_REPLACE:
-		patcherr = load(msg, patchPath, patchFmt, opts)
-	case tableaupb.Patch_PATCH_MERGE:
-		patchMsg := proto.Clone(msg)
-		// load msg from the "main" file
-		if err := load(msg, path, fmt, opts); err != nil {
-			return err
-		}
-		// load patchMsg from the "patch" file
-		if err := load(patchMsg, patchPath, patchFmt, opts); err != nil {
-			return err
-		}
-		patcherr = xproto.PatchMerge(msg, patchMsg)
-	default:
-		return xerrors.Errorf("unknown patch type: %v", patch)
+	patchMsg := proto.Clone(msg)

```

Comment:
This causes a big overhead which is not necessary if message patch option is `PATCH_NONE`

Reply:
Already handled previously